### ISSUE TITLE
Archive failing

### ISSUE THREAD

**Issue by rahulom** [forcedotcom/SalesforceMobileSDK-CordovaPlugin#608]
The xcode is throwing an error while archiving the build. Derived data error is throwing by the xcode. The application running fine in simulator and devices. 
I tried to remove an extra Images.xcassets from Build Faces->Copy bundle resources -But still the issue not resolved.

Cordova Info---

Cordova Packages:

    cli: 11.0.0
        common: 4.0.2
        create: 4.0.0
        lib: 11.0.0
            common: 4.0.2
            fetch: 3.0.1
            serve: 4.0.0

Project Installed Platforms:

    ios: 6.2.0

Project Installed Plugins:

    com.salesforce: 10.2.0
    cordova-plugin-device: 2.1.0
    cordova-plugin-file-opener2: 3.0.5
    cordova-plugin-file: 7.0.0
    cordova-plugin-inappbrowser: 5.0.0
    cordova-plugin-statusbar: 3.0.0

Environment:

    OS: macOS Monterey 12.6 (21G115) (darwin 21.6.0) x64
    Node: v16.17.0
    npm: 8.15.0

ios Environment:

    xcodebuild:
Xcode 14.0
Build version 14A309

Project Setting Files:

    config.xml:
<?xml version='1.0' encoding='utf-8'?>
<widget id="app.schoolpoort.com" version="1.0.0" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
    <name>appinfo</name>
    <description>appinfo</description>
    <author email="info@app.nl" href="https://appinfo.nl/">appinfo Team</author>
    
    <content src="index.html" />
    <allow-intent href="http://*/*" />
    <allow-intent href="https://*/*" />
    <preference name="scheme" value="app" />
    <preference name="hostname" value="localhost" />
    <platform name="ios">
        <icon height="57" src="resources/ios/icon/icon.png" width="57" />
        <icon height="114" src="resources/ios/icon/icon@2x.png" width="114" />
        <icon height="20" src="resources/ios/icon/icon-20.png" width="20" />
        <icon height="40" src="resources/ios/icon/icon-20@2x.png" width="40" />
        <icon height="60" src="resources/ios/icon/icon-20@3x.png" width="60" />
        <icon height="29" src="resources/ios/icon/icon-29.png" width="29" />
        <icon height="58" src="resources/ios/icon/icon-29@2x.png" width="58" />
        <icon height="87" src="resources/ios/icon/icon-29@3x.png" width="87" />
        <icon height="48" src="resources/ios/icon/icon-24@2x.png" width="48" />
        <icon height="55" src="resources/ios/icon/icon-27.5@2x.png" width="55" />
        <icon height="88" src="resources/ios/icon/icon-44@2x.png" width="88" />
        <icon height="172" src="resources/ios/icon/icon-86@2x.png" width="172" />
        <icon height="196" src="resources/ios/icon/icon-98@2x.png" width="196" />
        <icon height="216" src="resources/ios/icon/icon-108@2x.png" width="216" />
        <icon height="40" src="resources/ios/icon/icon-40.png" width="40" />
        <icon height="80" src="resources/ios/icon/icon-40@2x.png" width="80" />
        <icon height="120" src="resources/ios/icon/icon-40@3x.png" width="120" />
        <icon height="50" src="resources/ios/icon/icon-50.png" width="50" />
        <icon height="100" src="resources/ios/icon/icon-50@2x.png" width="100" />
        <icon height="60" src="resources/ios/icon/icon-60.png" width="60" />
        <icon height="120" src="resources/ios/icon/icon-60@2x.png" width="120" />
        <icon height="180" src="resources/ios/icon/icon-60@3x.png" width="180" />
        <icon height="72" src="resources/ios/icon/icon-72.png" width="72" />
        <icon height="144" src="resources/ios/icon/icon-72@2x.png" width="144" />
        <icon height="76" src="resources/ios/icon/icon-76.png" width="76" />
        <icon height="152" src="resources/ios/icon/icon-76@2x.png" width="152" />
        <icon height="167" src="resources/ios/icon/icon-83.5@2x.png" width="167" />
        <icon height="1024" src="resources/ios/icon/icon-1024.png" width="1024" />
        <splash height="1136" src="resources/ios/splash/Default-568h@2x~iphone.png" width="640" />
        <splash height="1334" src="resources/ios/splash/Default-667h.png" width="750" />
        <splash height="2688" src="resources/ios/splash/Default-2688h~iphone.png" width="1242" />
        <splash height="1242" src="resources/ios/splash/Default-Landscape-2688h~iphone.png" width="2688" />
        <splash height="1792" src="resources/ios/splash/Default-1792h~iphone.png" width="828" />
        <splash height="828" src="resources/ios/splash/Default-Landscape-1792h~iphone.png" width="1792" />
        <splash height="2436" src="resources/ios/splash/Default-2436h.png" width="1125" />
        <splash height="1125" src="resources/ios/splash/Default-Landscape-2436h.png" width="2436" />
        <splash height="2208" src="resources/ios/splash/Default-736h.png" width="1242" />
        <splash height="1242" src="resources/ios/splash/Default-Landscape-736h.png" width="2208" />
        <splash height="1536" src="resources/ios/splash/Default-Landscape@2x~ipad.png" width="2048" />
        <splash height="2048" src="resources/ios/splash/Default-Landscape@~ipadpro.png" width="2732" />
        <splash height="768" src="resources/ios/splash/Default-Landscape~ipad.png" width="1024" />
        <splash height="2048" src="resources/ios/splash/Default-Portrait@2x~ipad.png" width="1536" />
        <splash height="2732" src="resources/ios/splash/Default-Portrait@~ipadpro.png" width="2048" />
        <splash height="1024" src="resources/ios/splash/Default-Portrait~ipad.png" width="768" />
        <splash height="960" src="resources/ios/splash/Default@2x~iphone.png" width="640" />
        <splash height="480" src="resources/ios/splash/Default~iphone.png" width="320" />
        <splash height="2732" src="resources/ios/splash/Default@2x~universal~anyany.png" width="2732" />
    </platform>
</widget>

    package.json:
--- Start of Cordova JSON Snippet ---
{
  "platforms": [
    "ios"
  ],
  "plugins": {
    "com.salesforce": {},
    "cordova-plugin-device": {},
    "cordova-plugin-file-opener2": {},
    "cordova-plugin-file": {},
    "cordova-plugin-inappbrowser": {},
    "cordova-plugin-statusbar": {}
  }
}
--- End of Cordova JSON Snippet ---

**Comment by JeroenSchu** [comment]
any suggestion? @wmathurin

**Comment by rahulom** [comment]
Finally everything is working. I simply removed this plugin and implimented the custom Oauth flow.

**Comment by va1bhav** [comment]
Fix for this on Mobile SDK discussion forum 
https://trailhead.salesforce.com/trailblazer-community/feed/0D54S00000JgC8FSAV

**Comment by jake-moritz-trifecta** [comment]
I'm also running into Xcode archive issues with SDK 10.2.0, Xcode 14 + modern build system, anyone have a workaround for now?

**Comment by gkotula** [comment]
@jake-moritz-trifecta We are still working on incorporating the fix directly, but the link above (https://trailhead.salesforce.com/trailblazer-community/feed/0D54S00000JgC8FSAV) has some steps that has worked for others in the meantime.  Could you give that a try and see if those steps work for you as well?

**Comment by jake-moritz-trifecta** [comment]
> 

Thanks for the fast reply! I have already updated my build scripts to remove libCordova.a from linked binaries and remove CordovaLib from the target dependencies as mentioned in the link. This has allowed my local Xcode builds and archives to work properly. 

As part of my testing of these changes, I tried to upload my app's .ipa file to App Store Connect. I am getting these errors, which I haven't gotten before:

![image](https://user-images.githubusercontent.com/31135746/222837124-aa9e65a8-f2c6-49f4-b96d-68d635fa353a.png)
